src/Tickets/Commerce/Gateways/PayPal/Merchant.php

```diff
@@ -912,6 +912,27 @@ public function get_locale() {
 		 *
 		 * @param string $locale
 		 */
-		return apply_filters( 'tec_tickets_commerce_gateway_paypal_merchant_locale', $locale );
+		$locale = apply_filters( 'tec_tickets_commerce_gateway_paypal_merchant_locale', $locale );
```

Comment:
Should we first validate and then filter? That way the filter can override any problem we may have. I don't have a strong opinion on this, just raising the question.

Reply:
Not in this case. As we need to make sure that the final value we return is valid, filtering the value may allow users to supply invalid values.

Reply:
That will happen with most filters. For example, if you filter `wp_mail` and you input a wrong email address the function will fail, but it'll be a users problem. I was thinking more about flexibility and having the capacity to use a snippet in the case there's some problem coming up with this (so that we don't have to hotfix).